src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -3,3 +3,4 @@
 
 [assembly: AssemblyMetadataAttribute("Serviceable", "True")]
 [assembly: InternalsVisibleTo("dotnet.Tests")]
+[assembly: InternalsVisibleTo("Microsoft.DotNet.ProjectJsonMigration.Tests")]
```

Comment:
ProjectJsonMigration.Tests needs to use some of the utilities packaged with the dotnet-migrate in the EndToEnd tests. 

Perhaps another test project for EndToEnd tests would alleviate your concern here?